### DISCUSSION TITLE
更新通知（2017-08-27）

### DISCUSSION THREAD

**Discussion by SeaHOH** [SeaHOH/GotoX#42] [公告]
同步到 GoProxy GAE 服务端使用的新 API（r84），性能有所提升；

需要更新服务端版本，本项目 fork 的  GoProxy GAE 服务端对应版本为 r2；

同时因 API 名称相同，无法再兼容 GoAgent（请忽视3.2.x 之前版本）和 XX-Net 客户端。

---
服务端有更新，更新为 r4 对应 GoProxy GAE r85。

**Comment by caofengzi** [comment]
用p大的服務端可以不？另外你這個服務端多了個dll文件夾有什麼作用？

**Comment by Just2co** [comment]
@SeaHOH GAE服务端这么一升级到R84版本，很多基于GAE的旧客户端都失效了，现在能用的就最新的goproxy加上你这个GotoX了

**Comment by SeaHOH** [comment]
@caofengzi 
基本一样，区别可以看分支介绍，DLLs 是 Python 自带链接库。

@Just2co 
大概是吧，感觉我祸害了不少人。
不过还好，升级与否用户可以自己决定嘛。准备好了再升级也可以的。

**Comment by caofengzi** [comment]
@Just2co 這個使用最新客戶端還能使用。GoAgent-Always-Available

**Comment by SeaHOH** [comment]
顶楼信息更新，服务端 r4。

**Comment by hdoo** [comment]
没有windows版本的吗，我怎么找不到呢？

**Comment by caofengzi** [comment]
![20170831114433](https://user-images.githubusercontent.com/13650252/29905556-f1e91ef2-8e41-11e7-9dec-265068e93ddb.png)
